.github/actions/run-tests-in-container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Container with dependencies necessary to run unit-tests and code coverage
+
+# Build development environment
+# $ docker build . --tag note-arduino-test
+
+# Launch development environment (mount source root as /nes-rob/)
```

Comment:
nes-rob comment probably shouldn't be here.

Reply:
:robot: